code/_onclick/hud/hud.dm

```diff
@@ -95,17 +103,13 @@ var/datum/global_hud/global_hud = new()
 	var/show_intent_icons = 0
 	var/hotkey_ui_hidden = 0	//This is to hide the buttons that can be used via hotkeys. (hotkeybuttons list of buttons)
 
-	var/obj/screen/lingchemdisplay
-	var/obj/screen/lingstingdisplay
+	var/obj/screen/ling/chems/lingchemdisplay
+	var/obj/screen/ling/sting/lingstingdisplay
 
 	var/obj/screen/blobpwrdisplay

```

Comment:
I changed all mob health display to all be children of obj/screen/healths and moved healths from a mob var to be a hud var, this way we have one var instead of var/blobhealthdisplay , var/guardianhealthdisplay, var/deity_health_display, etc...